spec/models/stamp_duty_spec.rb

```diff
@@ -6,37 +6,37 @@
       expect(subject.price).to be_zero
     end
 
-    it 'sets second_home to false' do
-      expect(subject.second_home).to be_falsy
+    it 'sets buyer_type to standard by default' do
+      expect(subject.buyer_type).to eq'isNextHome'
```

Comment:
`expect(subject.buyer_type).to eq'isNextHome'` --> `expect(subject.buyer_type).to eq('isNextHome')`